core/src/main/java/com/kin/ecosystem/core/accountmanager/AccountManagerImpl.java

```diff
@@ -152,6 +159,7 @@ public void onResponse(Void response) {
 
 						@Override
 						public void onFailure(KinEcosystemException error) {
+							instance.error = error;
```

Comment:
also in switchAccount?

Reply:
@YohayBar not sure i understand why do you mean ?

Reply:
set error in onFailure within the switch account